app/src/main/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -239,7 +260,7 @@ class MessagingService : FirebaseMessagingService() {
                 }
                 val actionPendingIntent = PendingIntent.getBroadcast(
                     this,
-                    notificationAction.key.hashCode(),
+                    notificationAction.title.hashCode(),
```

Comment:
Can't this go wrong if I fire tow times "Motion detected" as title ? Can't we use time stamp + title or something ?

Reply:
Yep, good call.  I was only thinking in the context of a single notification.